src/handlers.ts

```diff
@@ -135,4 +138,207 @@ app.post("/store-token", async (c) => {
     });
 });
 
+app.get("/pong/t", async (c) => {
```

Comment:
can we move these to a different file, to have modularization.

Reply:
Done